src/databricks/labs/ucx/azure/access.py

```diff
@@ -3,6 +3,8 @@
 import uuid
 from dataclasses import dataclass
 from functools import partial
+import re
+from collections.abc import ValuesView
```

Comment:
``` suggestion
import re
import uuid
from collections.abc import ValuesView
from dataclasses import dataclass
from functools import partial
```

Reply:
@JCZuurmond Thanks. Code updated with this.